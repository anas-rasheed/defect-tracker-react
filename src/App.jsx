import React from 'react';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import { Provider } from 'react-redux';
import Header from './components/Header';
import store from './redux/store';
import Login from './components/Login';
// import Dashboard from './components/Dashboard';
import HomePage from './components/HomePage';
import AddDefect from './components/AddDefect';

const App = () => {
  return (
    <Provider store={store}>
      <Router>
        <Header />
        <Switch>
          <Route path='/' exact component={HomePage} />
          <Route path='/login' component={Login} />
          <Route path='/addDefect' component={AddDefect} />
        </Switch>
      </Router>
    </Provider>
  );
};

export default App;
