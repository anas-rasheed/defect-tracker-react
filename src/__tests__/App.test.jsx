import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('App component', () => {
  it('should render App Component', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
