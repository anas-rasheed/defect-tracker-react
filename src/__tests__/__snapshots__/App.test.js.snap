// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render Coming Soon Let's Create 1`] = `
<BrowserRouter>
  <Header />
  <h1>
    Coming Soon
  </h1>
  <h1>
    Let's create
  </h1>
</BrowserRouter>
`;
